examples/src/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto&display=swap"
```

Comment:
By using system font stack, we avoid FOUC and other issues. Apart from that, Google loses another tracking hit from every display/usage/fork of the repository.